service/src/test/java/bio/terra/workspace/service/workspace/WorkspaceStateUnitTest.java

```diff
@@ -0,0 +1,46 @@
+package bio.terra.workspace.service.workspace;
+
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import bio.terra.workspace.common.BaseUnitTest;
+import bio.terra.workspace.common.fixtures.WorkspaceFixtures;
+import bio.terra.workspace.db.WorkspaceDao;
+import bio.terra.workspace.db.exception.ResourceStateConflictException;
+import bio.terra.workspace.service.logging.WorkspaceActivityLogService;
+import bio.terra.workspace.service.policy.PolicyValidator;
+import bio.terra.workspace.service.workspace.model.Workspace;
+import java.util.UUID;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.mock.mockito.MockBean;
+
+public class WorkspaceStateUnitTest extends BaseUnitTest {
+  @MockBean private PolicyValidator mockPolicyValidator;
+  @MockBean private WorkspaceActivityLogService mockWorkspaceActivityLogService;
+
+  @Autowired private WorkspaceService workspaceService;
+  @Autowired private WorkspaceDao workspaceDao;
+
+  @Test
+  void createDeleteStateTest() throws Exception {
```

Comment:
In some cases it might be good option to combine functionalities into a single test - each test may* have overhead for setup. Given that we at times do not want them to run in parallel, test run times will eventually go up. Not a concern atm though. 
Might not make much of a difference either way here since the tests are pretty small and doesnt have much set up needed.

Reply:
i agree in the case of connected test. i only ask to split in the case of unit (e.g. no cloud calls)

Reply:
OK, I'll do it just because you asked :)